clients/cli/src/common.rs

```diff
@@ -71,3 +73,17 @@ pub async fn process_transaction(
         ))
     }
 }
+
+pub async fn get_mint(
```

Comment:
nit: this name could be a little more clear. What do you think about naming it to something like this?

```suggestion
pub async fn get_mint_for_token_account(
```